tests/acquisition/covidcast/test_database.py

```diff
@@ -73,32 +73,6 @@ def test_count_all_rows_query(self):
     self.assertIn('select count(1)', sql)
     self.assertIn('from `covidcast`', sql)
 
-  def test_get_data_stdev_across_locations_query(self):
-    """Query to get signal-level standard deviation looks sensible.
-
-    NOTE: Actual behavior is tested by integration test.
```

Comment:
Was this orphaned or is there a matching integration test which should also be removed/modified?

Reply:
I wasn't able to find any other references in the repo